metaflow/plugins/kfp/tests/sample_flows/static_branching.py

```diff
@@ -0,0 +1,60 @@
+from metaflow import FlowSpec, step
+
+
+class StaticBranching(FlowSpec):
```

Comment:
is this test subsumed by any of the foreach flow tests?

Reply:
Removing this test as it's not testing anything new.